README.md

```diff
@@ -14,14 +14,14 @@ Ensure you have:
 2. Move into the directory: `cd SeniorDesign2023BE\backend`
 3. Create a Virtual Environment: `python -m venv venv`
 4. Load the venv: `.\venv\Scripts\activate`
-5. Install all the packages: `pip install -r requirements.txt` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
+5. Install all the packages: `pip3 install -r ../requirements.txt --target=".\venv\Lib\site-packages" --upgrade` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
 6. Move into the src directory: `cd src`
 7. Run the server: `flask run` or `python app.py` (running the second command runs it in debug mode, so it will restart every time a file is changed)
 
 #### Other Times:
 1. Move into the main directory: `cd SeniorDesign2023BE\backend`
 2. Load the venv: `.\venv\Scripts\activate`
-3. Ensure you have all the packages: `pip install -r requirements.txt` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
+3. Ensure you have all the packages: `pip3 install -r ../requirements.txt --target=".\venv\Lib\site-packages" --upgrade` **IMPORTANT! IF YOU INSTALL OTHER PACKAGES ADD THEM TO "requirements.txt"**
 4. Move into the src directory: `cd src`
 5. Run the server: `flask run` or `python app.py` (running the second command runs it in debug mode, so it will restart every time a file is changed)
```

Comment:
flask run did not work for me (maybe because I don't have flask) but python app.py worked: 
![image](https://github.com/rgreenan55/SeniorDesign2023BE/assets/97375107/f5a59618-063b-4dcc-8f9d-5629ebf00196)

Reply:
yeah sometimes the flask command doesnt get added to your path, running it as the python app.py is good enough

There are 3 test files:
endpoints.py - To test the real endpoints (Real AI)
endpoints_test.py - To test the fake endpoints (fake AI)
data.py - To test that the data is loadable

the documentation doesnt need to change

Reply:
yeah sometimes the flask command doesnt get added to your path, running it as the python app.py is good enough

There are 3 test files:
endpoints.py - To test the real endpoints (Real AI)
endpoints_test.py - To test the fake endpoints (fake AI)
data.py - To test that the data is loadable

the documentation doesnt need to change